h/templates/activity/search.html.jinja2

```diff
@@ -26,7 +26,7 @@
   </div>
   <ul class="search-bucket-stats__val">
     {% for user in bucket.users %}
-      <li>{{ user.split(':')[1] }}</li>
+      <li>{{ user.split(':')[1].split('@')[0] }}</li>
```

Comment:
OK for the moment, in future we'll probably want to extract this out into a helper or pre-process `bucket.users` before rendering the result in the template.